src/structure/options.html.in

```diff
@@ -16,6 +16,10 @@
                         <input class="form-check-input" id="space-filling" type="checkbox" />
                         <label class="form-check-label" for="space-filling" title="show atoms as spheres">space filling</label>
                     </div>
+                    <div class="form-check form-switch">
```

Comment:
This should be hidden if all the shapes are spheres (in the same way we hide the checkbox for unitcell if there is no unit cell data)